.github/workflows/release.yml

```diff
@@ -55,7 +56,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          gh release upload --clobber "${{ github.ref_name }}" \
+          vsn=$(cat VERSION)
+          gh release upload --clobber "v${vsn}" \
             elixir-otp-${{ matrix.otp }}.zip \
```

Comment:
sorry if we already discussed it. I wonder if we should go with:

```
elixir-1.14.0-otp-24.zip
```

I think it is useful to put the version number in there. If so, we should change the nightly.yml accordingly too, I think we could go with: `elixir-main-otp-X.zip`.

Reply:
I remember that the previous pr use the pattern that you mentioned 

https://github.com/elixir-lang/elixir/pull/11522/files

Reply:
I get it. The file pattern was change to eliminate the `cat VERSION` [here](https://github.com/elixir-lang/elixir/commit/be1107884972ac433d7b8ddf943d99b7143725a4). 

If we are agree to bring this pattern back, i can open the PR to fix this. :)

Reply:
cc @josevalim

Reply:
It is fine to depend on `cat VERSION` for tags but outside of a tag it is not guaranteed to be exact.